cleanup/cleanup-bare-pods/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: cleanup-bare-pods
+spec:
+  steps:
+    - name: apply clusterrole
+      try:
+        - apply:
+            file: clusterrole.yaml
```

Comment:
@Jay179-sudo It seems you forgot to rename this as well to `cluster-role.yaml`.

PS:
Also the DCO signing